<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>I2C_VMWidget</class>
 <widget class="QWidget" name="I2C_VMWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="verticalSpacing">
    <number>-1</number>
   </property>
   <item row="1" column="0" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>620</width>
        <height>324</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="5" column="5">
        <widget class="QPushButton" name="SavePushButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="ApplyPushButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>175</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="hexDecComboBox">
         <property name="toolTip">
          <string>Select whether all field values are in hexadecimal or decimal.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>300</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="i2cAddressLabel">
         <property name="text">
          <string>I2C address (7-bit)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="AdditionalLinesPushButton">
         <property name="text">
          <string>Add more lines</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <layout class="QVBoxLayout" name="vmInstructionLayout"/>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QStackedWidget" name="i2cAddressStackedWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="decimalPage">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSpinBox" name="i2cDecimalAddressSpinBox">
             <property name="prefix">
              <string>0d</string>
             </property>
             <property name="maximum">
              <number>127</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="hexPage">
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
